edge-agent/docker/windows/arm32v7/Dockerfile

```diff
@@ -1,4 +1,4 @@
-ARG base_tag=1.0.5-windows-arm32v7
+ARG base_tag=1.1.0-windows-arm32v7
```

Comment:
I changed windows to be consistent that upgrade to netcore3.1 should start from 1.1.x.  I think we don't need to build Windows ARM32 base image for now, although image build still includes windows arm32 docker image but edgelet build doesn't.  Please confirm with Varun, if it is ok to remove Windows ARM32 from image build.  If so, no need to build Windows ARM32 base image, otherwise please build new docker image.